public_html/wp-content/plugins/wordcamp-speaker-feedback/includes/class-walker-feedback.php

```diff
@@ -96,20 +97,22 @@ protected function render_feedback( $comment, $depth, $args ) {
 					<?php render_feedback_comment( $comment ); ?>
 				</div><!-- .comment-content -->
 
-				<footer class="speaker-feedback__helpful <?php echo ( $comment->helpful ) ? 'is-helpful' : ''; ?>">
-					<span id="sft-helpful-<?php echo absint( $comment_id ); ?>">
-						<?php esc_html_e( 'Was this feedback helpful?', 'wordcamporg' ); ?>
-					</span>
-					<label>
-						<input
-							type="checkbox"
-							data-comment-id="<?php echo absint( $comment_id ); ?>"
-							aria-describedby="sft-helpful-<?php echo absint( $comment_id ); ?>"
-							<?php checked( $comment->helpful ); ?>
-						/>
-						<?php esc_html_e( 'Yes', 'wordcamporg' ); ?>
-					</button>
-				</footer>
+				<?php if ( in_array( get_current_user_id(), $session_speakers, true ) ) : ?>
```

Comment:
I think this should still be displayed, but it should be un-clickable, just in case anyone wants to style it (also so organizers know it should be there). I think if you remove the input when not a speaker, that should prevent the submission.

maybe with a note … somewhere… that only speakers can mark things as helpful?

Reply:
Fair enough. Fixed here: c16c8302